docs/support/faq/how-to-use-pouta-for-remote-desktop.md

```diff
@@ -0,0 +1,147 @@
+# How to use cPouta for Remote Desktop
+
+This chapter will go through how to set up a remote desktop with noVNC and
+ssh-tunnel on a non-GPU flavor in c- and ePouta. We are using noVNC because
+it allows us to use our browser to access the desktop. If you would like to have
+instruction how to use a GPU instance for rendering you can find instructions
+here [How to use cPouta GPU for rendering](how-to-use-cpouta-gpu-for-rendering.md)
+
+[TOC]
+
+## Preparations
+We will utilize the following technologies to install the remote desktop:
+
+  - A standard-flavor for example standard.medium 
+  - CentOS-7 images (In Pouta it has already the epel-repository installed)
+  - snapd to install the noVNC
+  - noVNC this allows to use our browser instead of a traditional VNC-client to view the desktop
+  - tigervnc-server as our VNC server.
+  -  Xfce as our Desktop environment.
+  -  ssh-tunnel so that the VNC server is not open to the internet. This is very
+important.
+
+## Create and access your instance for remote desktop
+
+1. Launch an standard-flavor instances with the CentOS-7 image.
+2. Attach a floating IP to the instance.
+3. In the security rules allow ingress ssh (port 22).
+4. We will ssh into the instance with this command to create a ssh-tunnel 
+tunnel. Note that the port 2001 is the one that you will use with the browser later.
+
+```
+ssh -L2001:localhost:6081 cloud-user@YOUR-FLOATING-IP
+```
+
+`-L2001:localhist:6081` meant that will be able access port 6081 on the server
+from our computers local port 2001.
+
+## Install the required software
+
+In this example we are using Xfce for our Desktop Environment you want to use
+another Desktop environment you will probably  need to modify the
+xstartup script.
+
+```
+sudo yum group install Xfce
+```
+
+Install snapd for noVNC and tigervnc for the vncserver (epel-release should already be installed)
+
+```
+sudo yum install -y epel-release snapd tigervnc-server
+```
+
+Start the snapd service so that we can install the noVNC
+
+```
+sudo systemctl start snapd
+```
+
+Install noVNC
+
+```
+sudo snap install novnc
+```
+
+## Configure our software
+
+Create a base configure for the user you want to be able to use the remote desktop.
+
+```
+mkdir ~/.vnc
+touch /home/cloud-user/.Xauthority
+```
+
+Configure a resolution you would like to use. 1920x1080 is a common resolution
+but this have been tested to work with an resolution as big as 3840x2160 .
+
+```
+echo "geometry=1920x1080" > ~/.vnc/config
+```
+
+Configure the startup script.
+
+```
+echo '#!/bin/sh
+unset SESSION_MANAGER
+unset DBUS_SESSION_BUS_ADDRESS
+xrdb $HOME/.Xresources
+startxfce4 --display=:1 &' > /home/cloud-user/xstartup
+```
+
+Add execution permission if you do not do this you will get a blank screen.
+
+```
+chmod +x /home/cloud-user/xstartup
+```
+
+## Starting your remote desktop
+
+Start vncserver
+
+```
+vncserver :1
```

Comment:
does this steal your shell or push to a background process?

Reply:
I guess background.